### PR TITLE
Update normalize scss to v6

### PR DESCRIPTION
**Pull request by t-kelly**
### What are you trying to accomplish with this PR?
Fixes https://github.com/Shopify/slate/issues/121

For maintainers:
- [x] I have :tophat:'d these changes.
- [ ] I have bumped the `package.json` version in a separate PR, if applicable.

### CODE REVIEW

**Comment by chrisberthe** [comment]
Doesn't seem like it, but this wouldn't have any unforeseen repercussions with any other Slate styling would it?

File: src/styles/global/normalize.scss [hunk 1]
```diff
@@ -1,163 +1,171 @@
-/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v6.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (huguestennier) [inline src/styles/global/normalize.scss]: v7.0.0 is out - any reason we use v6.0.0 instead?

Comment (chrisberthe) [inline src/styles/global/normalize.scss]: Yeah, seems to just be some small tweaks: https://github.com/necolas/normalize.css/commit/b4a8fdaf8321093b1e3cad612fb76cf2b7d3275d

Should be fine to upgrade to v7.0.0.

**Comment by t-kelly** [comment]
@huguestennier updated

**Pull request merged**